### DISCUSSION TITLE
How do I set the language for swipe/gestures?

### DISCUSSION THREAD

**Discussion by joaocandre** [florisboard/florisboard#1481] [Q&A] [answered]
I have two languages setup on my phone (running LineageOS 18.1) and two keyboard layouts setup in Florisboard, but typing with swipe only English words are auto-suggested. Is there a way to have different language setup for swipe for each keyboard layout?

**Comment by patrickgold** [comment, marked as answer]
At the moment no, this is due to the fact that glide typing and word suggestions work with the same dictionaries and word suggestions (the current implementation) only supports English (US). After the preference rework the next big project will be the implementation of proper word suggestions for all languages, then glide typing will be available in different languages too.

**Reply by joaocandre** [reply]
Thanks. Is there any estimate as to when exactly will it become available?

**Reply by patrickgold** [reply]
Not exactly, but Q2 this year is the time when word suggestions are planned to work and with that also glide typing locales.
